### PR TITLE
Add FreezeLog functionality

### PR DESCRIPTION
**Pull request by YoshihitoAso**
- Close: #582 
- Add following APIs.
  - POST: `/freeze_log/accounts`
    - Creation of an account (private key) for logging. 
    - The private key is stored in the same format as the issuer key.
  - GET: `/freeze_log/accounts`
    - List all accounts.
  - DELETE: `/freeze_log/accounts/{account_address}`
    - Delete a logging account.
  - POST: `/freeze_log/accounts/{account_address}/eoa_password`
    - Change EOA kefile password.
  - POST: `/freeze_log/logs`
    - Record new log message.
  - POST: `/freeze_log/logs/{log_index}`
    - Update log message.
  - GET: `/freeze_log/logs/{log_index}`
    - Retrieve stored log.

### CODE REVIEW

File: app/routers/freeze_log.py [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+        select(FreezeLogAccount)
+        .where(FreezeLogAccount.account_address == req.account_address)
+        .limit(1)
+    ).first()
+    if log_account is None:
+        raise HTTPException(status_code=404, detail="account is not exists")
+
+    # Authentication
+    eoa_password = (
+        E2EEUtils.decrypt(req.eoa_password) if E2EE_REQUEST_ENABLED else req.eoa_pass
```
Comment (purplesmoke05) [inline app/routers/freeze_log.py:302]:
```suggestion
        E2EEUtils.decrypt(req.eoa_password) if E2EE_REQUEST_ENABLED else req.eoa_password
```

Comment (YoshihitoAso) [reply app/routers/freeze_log.py:304]: Thank you for pointing that out.
Added a test case for the no encryption option as well.

Comment (purplesmoke05) [reply app/routers/freeze_log.py]: I completely forgot about the `update_log`. Pushed a trivial commit. https://github.com/BoostryJP/ibet-Prime/pull/587/commits/508d9a7a357f99710cb5e5516b39dc923eb1ddbe

**Review by purplesmoke05 — APPROVED** [review]
LGTM

**Pull request merged**